tmt/steps/execute/__init__.py

```diff
@@ -93,7 +93,7 @@ class ScriptCreatingFile(Script):
         Path("/usr/local/bin/rstrnt-report-result"),
         Path("/usr/local/bin/rhts-report-result")],
     related_variables=[],
-    created_file="restraint-result"
+    created_file="restraint-results.yaml"
```

Comment:
`tmt-report-results.yaml` was proposed on call.

Reply:
Fixed in 0ed9f004.